README.md

```diff
@@ -1,11 +1,12 @@
 # Kamzzy Blog
-> An application to make budgeting easier
+> An application that make it easy for you to keep track of your expenditures and plan your finances better.
 
```

Comment:
- [ ] It would be better if we fully address the comments made by the previous code reviewer. The link to the mit.md is broken because the file is not present. Kindly add the `mit.md` file in the root directory